README.md

```diff
@@ -1,4 +1,28 @@
-# Behind-the-Scenes Readme
-This file will be the landing page for the GitHub repository, but will not be surfaced in the GitBook. Use it to outline the lesson plan and provide any other instructor-facing information. Note that this page will not actually be private or secure; it is simply separated from the GitBook.
+# Hyland JavaScript Academy @ UCS
+This repository contains material for Hyland's continued partnership with Urban Community School.
 
-[Click here for information about this template repository.](BOOKTEMPLATEREADME.md)
\ No newline at end of file
+## Learning Objectives
+In this academy, students will use JavaScript to create interactive websites. They will practice applying essential programming concepts like variables, output, input, and conditional statements to build unique web applications.
+
+_Note: In order to successfully participate, students must have solid skills in HTML & CSS (covered by Hyland Coding Academy in previous trimesters)._
+
+## Outline
+Here is an outline of the proposed lessons (subject to change as the trimester progresses).
+
+1. **HTML and CSS Review** -- _Refresh some memories. See: [Web 102 - HTML & CSS Review](https://github.com/hytechclub/web-102/tree/master/HtmlCssReview)_
+1. **Introduction to JavaScript** -- _If HTML is the body and CSS is the clothes, JavaScript is the mind. See: [Web 102 - Introduction to JavaScript](https://github.com/hytechclub/web-102/tree/master/IntroToJS)_
+1. **Variables and Data Types** -- _Learn the building blocks of programming. See: [Web 102 - Data Types](https://github.com/hytechclub/web-102/tree/master/DataTypes)_
+1. **Buttons** -- _Make your web page interactive. See: [Web 102 - Buttons & Functions](https://github.com/hytechclub/web-102/tree/master/Buttons)_
+1. **Conditionals** -- _Make your web page smart. See: [Web 102 - Conditionals](https://github.com/hytechclub/web-102/tree/master/Conditionals)_
+1. **Review and Check-ins**
```

Comment:
I don't know if we'll actually be able to do check-ins like we do in Hy-Tech Club, but having a week around this time dedicated to review could be really helpful.